strawberry_django/mutations/mutations.py

```diff
@@ -28,6 +30,7 @@
 )
 
 _T = TypeVar("_T")
+settings = strawberry_django_settings()
```

Comment:
Ditto

Reply:
I was thinking,
Maybe it's redundant here?
Anyway `key_attr` is propagating to [`DjangoMutationCUD`](https://github.com/strawberry-graphql/strawberry-graphql-django/pull/446/files#diff-c5989633bdbcc1fe4c92d5b3bb2b37ce9bbe1dc3d793adea4552d94caadd89adR178) and there it will be handled for Update and Delete right?